### PR TITLE
PIL-1011: Feature Onboarding flag

### PR DESCRIPTION
**Pull request by divyaBacancy**

### CODE REVIEW

File: src/actions/appActions.js [hunk 1]
```diff
@@ -76,6 +76,10 @@ import { getWalletFromStorage } from 'utils/wallet';
 // actions
 import { getTranslationsResourcesAndSetLanguageOnAppOpenAction } from 'actions/localisationActions';
 import { setWbtcPendingTxsAction } from './exchangeActions';
+import { REMOTE_CONFIG } from '../constants/remoteConfigConstants';
+
+//firebase
```
Comment (IAmKio) [inline src/actions/appActions.js:81]:
```suggestion
// firebase
```

File: src/actions/appActions.js [hunk 2]
```diff
@@ -236,7 +242,7 @@ export const initAppAndRedirectAction = () => {
           },
         }),
       };
-    } else {
+    } else if(feautureOnboarding) {
```
Comment (IAmKio) [inline src/actions/appActions.js:245]:
```suggestion
    } else if (feautureOnboarding) {
```

Do you have code linting enabled on your editor?

**Review by IAmKio — DISMISSED** [review]

File: src/actions/appActions.js (hunk 2 above)
Comment (divyaBacancy) [reply src/actions/appActions.js:245]: yes

**Comment by IAmKio** [comment]
@divyaBacancy could you please sign the CLA please? It looks like you may have authored with three different accounts, all three accounts need to sign the agreement.

File: src/actions/appActions.js (hunk 1 above)
Comment (divyaBacancy) [reply src/actions/appActions.js]: > @divyaBacancy could you please sign the CLA please? It looks like you may have authored with three different accounts, all three accounts need to sign the agreement.

@IAmKio I have signed the CLA for divyaBacancy account. Only this account is needed will avoid working from other account(divya-svmx). admin is not my account.

File: src/actions/onboardingActions.js [hunk 3]
```diff
@@ -311,8 +312,10 @@ export const finishOnboardingAction = (retry?: boolean, recoveryData?: Object) =
 
     await dispatch(getTutorialDataAction());
 
+    const feautureOnboarding = firebaseRemoteConfig.getString(REMOTE_CONFIG.FEATURE_ONBOARDING);
```
Comment (mdjastrzebski) [inline src/actions/onboardingActions.js:315]:
```suggestion
    const enableOnboarding = firebaseRemoteConfig.getString(REMOTE_CONFIG.FEATURE_ONBOARDING);
```

**Review by mdjastrzebski — COMMENTED** [review]
Minor naming suggestion

**Comment by mdjastrzebski** [comment]
@divyaBacancy the CLA/email issue could be to your git client configuration, where the username & email seem to be set to other values then your GitHub account (admin, @divya-svmx). Please make sure the username and email matches your GitHub credentials. Then you'll probably need to do some rebase/force push to clear the existing commit entries with wrong user info.

**Comment by divyaBacancy** [comment]
@mdjastrzebski I will be using divyaBacancy only and that is configured.

File: src/actions/onboardingActions.js (hunk 3 above)
Comment (divyaBacancy) [reply src/actions/onboardingActions.js]: @mdjastrzebski Thank you for reviewing.The changes have been done.

**Review by mdjastrzebski — APPROVED** [review]
LG!

**Pull request merged**